frontend/src/features/contact/list/ContactListView.tsx

```diff
@@ -0,0 +1,74 @@
+import { useApiContacts } from 'hooks/pims-api/useApiContacts';
+import { IContactSearchResult } from 'interfaces/IContactSearchResult';
+import { useCallback } from 'react';
+import { IoMdPersonAdd } from 'react-icons/io';
+import { toast } from 'react-toastify';
+
+import { useSearch } from '../../../hooks/useSearch';
+import { IContactFilter } from '../interfaces';
+import { ContactSearchResults } from './ContactSearchResults/ContactSearchResults';
+import ContactFilter, { defaultFilter } from './filter/ContactFilter';
+import * as Styled from './styles';
+
+/**
+ * Component that displays a list of leases within PSP as well as a filter bar to control the displayed leases.
+ */
+export const ContactListView = () => {
+  const { getContacts } = useApiContacts();
+  const {
+    results,
+    filter,
+    sort,
+    error,
+    currentPage,
+    totalPages,
+    pageSize,
+    setFilter,
+    setSort,
+    setCurrentPage,
+    setPageSize,
+    loading,
```

Comment:
if useSearch returns a typed object I think it would be better to use that one instead of creating a new one

Reply:
I mean the {...} is creating an object on the fly. Is the hook response type not sufficient?

Reply:
Destructuring is a JS best practice. It is syntactic sugar for:

```tsx
const obj = useSearch(...)
const blah = obj.blah

// use blah wherever instead of obj.blah

```

Reply:
The hook type is sufficient but code is clearer when we refer to "pageSize" instead of "hookResponse.pageSize"